src/main/kotlin/com/exactpro/th2/common/message/MessageUtils.kt

```diff
@@ -336,6 +337,13 @@ fun getSessionAliasAndDirection(anyMessage: AnyMessage): Array<String> = when {
     else -> unknownLabels
```

Comment:
Please throw an exception instead of using plug

Reply:
done